src/common/metadata.c

```diff
@@ -117,6 +118,31 @@ const char *dt_metadata_get_key(const uint32_t keyid)
     return NULL;
 }
 
+const char *dt_metadata_get_subkey(const uint32_t keyid)
+{
+  if(keyid < DT_METADATA_NUMBER)
+  {
+    char *t = g_strrstr(dt_metadata_def[keyid].key, ".");
```

Comment:
We should probably guard against t begin null here.

Reply:
:), that's the constant list... OK.